data/json/terrain.json

```diff
@@ -1660,6 +1661,7 @@
         "type" : "terrain",
         "id" : "t_door_locked_alarm",
         "name": "closed wood door",
+        "description": "A standard wooden door and doesn't look like the expensive, more resilient wood types.  It'd probably burn easily, too.  This one has an extra keyhole, so it's likely locked.  It could be pried open or the lock picked.",
```

Comment:
> A standard wooden door and doesn't look like the expensive, more resilient wood types

"and" should be "that," since the door is the subject.

Also, what are the expensive, more resilient wood types referenced here?